iotdb-protocol/thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -1553,5 +1555,11 @@ service IConfigNodeRPCService {
 
   /** Get throttle quota information */
   TThrottleQuotaResp getThrottleQuota()
+
+  // ======================================================
+  // Table
+  // ======================================================
+
+  common.TSStatus createTable(binary tableInfo)
```

Comment:
is this interface used for write auto create? If so, just updating total tabel info each time will have problems in concurrent case?

Reply:
No. This is just the interface for executing manual table creation.